modules/provision/container.go

```diff
@@ -61,7 +61,7 @@ func ContainerProvision(ctx context.Context, reservation Reservation) (interface
 		Msg("deploying network")
 
 	networkMgr := stubs.NewNetworkerStub(GetZBus(ctx))
-	ns, err := networkMgr.Namespace(modules.NetID(config.Network.NetwokID))
+	ns, err := networkMgr.Join(reservation.ID, modules.NetID(config.Network.NetwokID))
```

Comment:
We should also return the IP allocated in the namespace to the caller, cause provisiond will have to send this information back to the BCDB at some point.

Reply:
True. Will do.